apps/site/assets/ts/schedule/components/line-diagram/SingleStop.tsx

```diff
@@ -19,6 +20,7 @@ interface Props {
   isOrigin?: boolean;
   isDestination?: boolean;
   liveData?: LiveData;
+  query?: string;
```

Comment:
Naming this `searchQuery` I think would make it clearer what it's referring to; since the functionality is part of a different component, it's not obvious what "query" means in this context (it could refer to the `query` part of the page URL, for example).